packages/frontend/src/ee/features/customRoles/components/ScopeSelector/index.tsx

```diff
@@ -0,0 +1 @@
+export { ScopeSelector } from './ScopeSelector';
```

Comment:
what is this for ?

Reply:
This is the component that powers finding scopes to assign. Exporting it here for easier importing.